robot/EDA/resources/EDA.py

```diff
@@ -381,3 +413,196 @@ def get_namespace_prefix(self, name):
             return parts[0] + '__'
         else:
             return ''
+
+
+    def close_all_tabs(self):
+        """ Gets the count of the tabs that are open and closes them all """
+        locator = eda_lex_locators["close_tab"]
+        count = int(self.selenium.get_matching_xpath_count(locator))
+        if count == 0:
+            return
+        else:
+            for i in range(count):
+                self.selenium.wait_until_element_is_visible(locator)
+                self.selenium.get_webelement(locator).click()
+
+
+    def click_on_element_if_exists(self, path, *args, **kwargs):
+        """ Clicks on the element if it exists
+            by building a locator using the path and args
+            but the keyword will not fail in case the element doesn't exist
+        """
+        main_loc = self.get_eda_locator(path, *args, **kwargs)
+        if self.check_if_element_exists(main_loc):
+            self.selenium.click_element(main_loc)
+
+    def convert_time_to_UTC_timezone(self, my_time):
+        """ Converts the given datetime to UTC timezone
+            my_time should be in the format %Y-%m-%d %H:%M:%S
+        """
+        my_time_format = datetime.datetime.strptime(my_time, "%Y-%m-%d %H:%M:%S.%f")
+        my_time_local = pytz.timezone("America/Los_Angeles").localize(my_time_format, is_dst=None)
+
+        my_time_utc = my_time_local.astimezone(pytz.utc)
+        return datetime.datetime.strftime(my_time_utc, "%Y-%m-%dT%H:%M:%S.%f")[:-3] + "Z"
+
+    def format_all(self, loc, value):
+        """ Formats the given locator with the value for all {} occurrences """
+        count = 0
+        for s in loc:
+            if s is '{':
+                count += 1
+
+        if count == 1:
+            return loc.format(value)
+        elif count == 2:
+            return loc.format(value, value)
+        elif count == 3:
+            return loc.format(value, value, value)
+
+    def get_instance_url(self, org=None):
```

Comment:
This doesn't appear to be used anywhere. You should delete it.

Reply:
Removed this code.